README.md

```diff
@@ -456,23 +456,22 @@ ConfigureMyObject(
     downstreamDependencies: originalObjectDependencies.Where(x => x.Direction == Direction.Downstream));
 ```
 
-### [F.3.4] ✔️ **CONSIDER** Splitting long chains of dotted method/property invocations
+### [F.3.4] ✔️ **DO** Split chains of method invocations with lambda expressions 💻
 
-If you invoke a chain of methods and/or properties and that single line becomes very long, split each invocation onto its own line. Use well-named local variables if needed.
+If you invoke a chain of methods with lambda expressions, split each invocation (except the first one) onto its own line. Use well-named local variables if needed.
 
 There is no hard limit on line length. Use judgement to keep code easy to read and diff. See [[F.3.1] ❌ **AVOID** Adding excessive complexity in compound statements](#%5Bf.3.1%5D-%3Ax%3A-**avoid**-adding-excessive-complexity-in-compound-statements).
 
 Indent dotted invocations one level deeper than the root object. Place the dot for each invocation at the beginning of that line. 
 
 ```csharp
 // Bad
-var distinctSelectedSoftware = selectedSoftwareContents.GroupBy(software => software.AliasName).Select(g => g.First()).Distinct();
+var distinctSoftware = softwareContents.GroupBy(software => software.AliasName).Select(g => g.First()).First(x => x == "Distinct");
```

Comment:
Why did you have to change the // Bad to change the Distinct() to a First? Was it just to have a third lambda? I think we can update this example to be a more realistic case. I'm actually curious to see what the recommendation is there for it. I know that this one is going to get a little contentious otherwise

Reply:
Based on our discussion, I updated the example to a more realistic case.

Reply:
I don't see your update. Am I missing something? I still see the old example

Reply:
ah it now says outdated! It should refresh soon